aldryn_jobs/forms.py

```diff
@@ -133,7 +133,7 @@ def is_edit_action(self):
 
     def validate_field_uniqueness_with_app_config(self, field_name,
                                                   error_message):
-        field = self.cleaned_data[field_name]
+        field = self.cleaned_data.get(field_name, '')
```

Comment:
if the .get() returns an empty string, does it make sense for the filter below to filter by that?